### PR TITLE
generate: Fixed issue with prefixing with modules when not needed

### PR DESCRIPTION
**Pull request by xescugc**
TF dependencies are prefixed already with the full path to the resource so it was not needed for us to prefix it with more information
as it was causing double prefixes in cases in which it was already prefixed

### CODE REVIEW

File: generate/state_test.go [hunk 1]
```diff
@@ -35,9 +34,45 @@ func TestFromState(t *testing.T) {
 		require.NoError(t, err)
 
 		_, _, err = generate.FromState(src, generate.Options{Clean: true, Connections: true, ExternalNodes: true})
-		fmt.Println(err)
 		assert.True(t, errors.Is(err, errcode.ErrInvalidTFStateFileMissingResourceID))
 	})
+	t.Run("MultyModule", func(t *testing.T) {
```
Comment (xlr-8) [inline generate/state_test.go:39]: multi* (plus repetition)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 ## [Unreleased]
 
+### Fixed
+
+- Multiple modules connecting between them now works correctly
+  ([Issue #120](https://github.com/cycloidio/inframap/pull/120))
```
Comment (xlr-8) [inline CHANGELOG.md:6]: Wrong link it seems, but good redirection

**Review by xlr-8 — CHANGES_REQUESTED** [review]
Would that work with module of module?

File: CHANGELOG.md (hunk 2 above)
Comment (xescugc) [reply CHANGELOG.md:6]: A yeh forgot to update it once pushed

**Comment by xescugc** [comment]
> Would that work with module of module?

Yes, the logic that was wrong was that I thought the resources on the `dependencies` key did not have the module prefix, but based on other examples I found out that they always have so no need to prefix them. And at the end TF already has the prefix correctly AFAIK.

**Comment by xescugc** [comment]
Pushed a new change to always display the `module` preix of the resource instead of trying to guess if it was needed or not.

Done.

**Review by xlr-8 — APPROVED** [review]

**Pull request merged**